crypto/verifier.go

```diff
@@ -31,8 +31,13 @@ import (
 	"github.com/google/trillian/crypto/sigpb"
 )
 
-// ErrVerify occurs whenever signature verification fails.
-var ErrVerify = errors.New("signature verification failed")
+var (
+	errVerify = errors.New("signature verification failed")
```

Comment:
In Go, the convention is that you can test errors like so `if err == ErrVerify` which is quite nice. Now in this particular case, if you don't think external libraries should be doing anything special with this error condition, then it's perfectly fine to hide it for better data hiding. 

Thanks for this change. Keep it as `errVerify`